src/index.html

```diff
@@ -14,9 +14,13 @@ <h3>Today's To Do</h3>
         </li>
         <li>
           <input type="text" id="add-list" placeholder="Add to your list">
-          <i class="fa-solid fa-right-to-bracket"></i>        
+          <i class="fa-solid fa-right-to-bracket add-btn" id="add-btn"></i>        
         </li>
+        <li class="clear-completed"><span class="clear-all-completed">Clear all completed</span></li>
       </ul>
+      
+
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use a `button` tag on the `Clear all completed` for good semantics tags. Also, remember to remove the empty code lines to make your codebase more liable and professional.